src/components/MenuItem.js

```diff
@@ -212,12 +221,18 @@ const MenuItem = (props) => {
                     </View>
                 </>
             )}
-        </Pressable>
+        </PressableWithSecondaryInteraction>
     );
 };
 
 MenuItem.propTypes = propTypes;
 MenuItem.defaultProps = defaultProps;
 MenuItem.displayName = 'MenuItem';
 
-export default MenuItem;
+const MenuItemWithWindowDimensions = withWindowDimensions(React.forwardRef((props, ref) => (
```

Comment:
That is a weird bug. 
Thanks for coming up with a fix so fast, but I wonder why this is happening and how could we fix this without applying workarounds. 
I'll try to investigate this further a bit before we can merge this

Reply:
Sounds good. I have an additional fix incoming for this issue: https://github.com/Expensify/App/issues/17656. Just need to verify the details first.